backend/src/test/java/com/redhat/cloud/notifications/recipients/rbac/AuthRequestFilterTest.java

```diff
@@ -17,7 +17,7 @@
 @QuarkusTest
 public class AuthRequestFilterTest {
 
-    private static final String testToken = "{\"approval\":{\"secret\":\"123\"},\"advisor\":{\"secret\":\"456\"},\"notifications\":{\"secret\":\"789\"}}";
+    private static final String testToken = "{\"approval\":{\"secret\":\"123\"},\"advisor\":{\"alt-secret\":\"456\"},\"notifications\":{\"secret\":\"789\"}}";
```

Comment:
With this change, the stage failure is reproduced during tests:

```
2021-09-02 16:15:07,586 ERROR [com.red.clo.not.rec.rba.AuthRequestFilter] (main) Unable to load Rbac service to service secret map, defaulting to empty map: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "alt-secret" (class com.redhat.cloud.notifications.recipients.rbac.AuthRequestFilter$Secret), not marked as ignorable (one known property: "secret"])
 at [Source: (String)"{"approval":{"secret":"123"},"advisor":{"alt-secret":"456"},"notifications":{"secret":"789"}}"; line: 1, column: 55] (through reference chain: java.util.LinkedHashMap["advisor"]->com.redhat.cloud.notifications.recipients.rbac.AuthRequestFilter$Secret["alt-secret"])
```

Reply:
how did it happen that the field name changed? Was it "alt-secret" the whole time and we just used the wrong field name from the beginning? Or did they change it and we didn't know about it?